niworkflows/anat/mni.py

```diff
@@ -48,7 +54,7 @@ class RobustMNINormalizationInputSpec(BaseInterfaceInputSpec):
                                       desc='template resolution')
     generate_report = traits.Bool(
         False, usedefault=True, desc='enable report generation')
-    html_report = File('viz-report.html', usedefault=True, desc='visual report file')
+    html_report = File('viz-report.svg', usedefault=True, desc='visual report file')
```

Comment:
Nope, this is the deal: the generated svg is standalone and works fine without anything else. I comment on why this is interesting below.

BTW, I would suggest to change the ```html_report``` by ```out_report``` which is more nipype-like or ```report_file``` which is also somewhat nipype-like and does not convey that the output is html always.